README.md

```diff
@@ -3,25 +3,24 @@
 
 <img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
 
-**Turn a leveldown iterator into a readable stream**
+> Turn a leveldown iterator into a readable stream
 
 [![Build Status](https://travis-ci.org/Level/iterator-stream.png)](https://travis-ci.org/Level/iterator-stream) [![Greenkeeper badge](https://badges.greenkeeper.io/Level/iterator-stream.svg)](https://greenkeeper.io/)
 
 ## Example
 
 ```js
-var iteratorStream = require('level-iterator-stream');
-var leveldown = require('leveldown');
-
-var db = leveldown(__dirname + '/db');
-db.open(function(err){
-  if (err) throw err;
-
-  var stream = iteratorStream(db.iterator());
-  stream.on('data', function(kv){
-    console.log('%s -> %s', kv.key, kv.value);
-  });
-});
+const iteratorStream = require('level-iterator-stream')
+const leveldown = require('leveldown')
+const db = leveldown(__dirname + '/db')
```

Comment:
can we keep the blank line between require statements and logic?

Reply:
Of course!